src/aatoolbox/utils/raster.py

```diff
@@ -266,22 +266,25 @@ def _check_coords_inverted(self):
         return lon[0] > lon[-1], lat[0] < lat[-1]
 
     def change_longitude_range(
-        self, inplace: bool = False
+        self, hundredeightyrange: bool = True, inplace: bool = False
```

Comment:
Could we change the argument here to be something like `range_180`?

Reply:
Or even thinking about it, should it be `to_range_180` so it's clear it's about destination range?

Reply:
ha  much better, my brain wasn't working there